tests/lib/testers/no-test-runners.js

```diff
@@ -0,0 +1,31 @@
+/**
+ * @fileoverview Tests for RuleTester without any test runner
+ * @author Weijia Wang <starkwang@126.com>
+ */
+"use strict";
+
+/* global describe, it */
+/* eslint-disable no-global-assign*/
+const assert = require("assert"),
+    RuleTester = require("../../../lib/testers/rule-tester");
+const tmpIt = it;
+const tmpDescribe = describe;
+
+it = null;
+describe = null;
+
+const ruleTester = new RuleTester();
```

Comment:
I'd feel better if this logic (from here to the end of the assertion) were wrapped in a try/finally so that the globals are definitely reassigned even if an exception is thrown.

Reply:
I've added the try/finally : )